app/views/needs/show.html.haml

```diff
@@ -48,7 +48,10 @@
       .cell.small-12.medium-9.large-10.shift{ class: shift.user_id ? 'secondary claimed' : 'success unclaimed' }
         .grid-x.grid-padding-x.grid-padding-y
           .cell.text-center
-            %strong= shift.start_at.strftime('%I:%M%p')
+            - if policy(@need).edit?
```

Comment:
You could reduce duplication like this:

```haml
%strong
  = shift.start_at.strftime('%I:%M%p')
  - if policy(@need).edit?
    = " - #{shift.user&.to_s}"
```

I think that'd work, but you should try it out as I have not executed it. :-)

Reply:
No problem!